changelog/bugfixes/2024-06-25-flatcar-install-custom-key.md

```diff
@@ -0,0 +1 @@
+- Fixed bad usage of gpg that prevented flatcar-install from being used with custom signing keys ([Flatcar#1471](https://github.com/flatcar/flatcar/pull/1471))
```

Comment:
This should have rather been `issues` instead of `pull` but whatever, redirects will do the trick. ;)
```suggestion
- Fixed bad usage of gpg that prevented flatcar-install from being used with custom signing keys ([Flatcar#1471](https://github.com/flatcar/Flatcar/issues/1471))
```